docs/build_commands.rst

```diff
@@ -368,6 +368,19 @@ The following ``pip`` package specification formats are supported:
 * The ``git+`` and ``hg+`` links (the git and mercurial are installed as build
   dependency automatically)
 
+  .. note::
+     In Ubuntu you may also need to install ``ca-certificates`` package
+     to be able to install ``git+/hg+`` python packages:
```

Comment:
Not sure who's bug this is.
In ubuntu `pip` exits with following error:
`Problem with the SSL CA cert (path? access rights?)`